opm/autodiff/BlackoilModelBase.hpp

```diff
@@ -338,12 +365,10 @@ namespace Opm {
             return static_cast<const Implementation&>(*this);
         }
 
-        // return true if wells are available in the reservoir
-        bool wellsActive() const { return wells_active_; }
```

Comment:
I think these three functions should still be implemented in the model class, forwarding to the std_wells_.